docs/src/denudation/empirical.md

```diff
@@ -1,32 +1,43 @@
 
-<img width="433" alt="image" src="https://github.com/MindTheGap-ERC/CarboKitten.jl/assets/64159957/29da37f3-0e14-479f-8ed3-a8b5adc052a5">
+<img width="433" alt="image" src="docs\src\fig\Precipitation-Denudation.png">
 
-*Fig 1. The relationship between MAP (mean precipitation per year, mm/y) and denudation rates (mm/ky)*
+*Fig 1. The relationship between MAP (mean precipitation per year, mm/y) and denudation rates (mm/ky)* 
 
-<img width="370" alt="image" src="https://github.com/MindTheGap-ERC/CarboKitten.jl/assets/64159957/1bd9e2ae-d4d7-4611-a8b8-83459222022f">
+<img width="370" alt="image" src="docs\src\fig\Slope-Denudation.png">
 
-*Fig 2. The relationship between the curvature (i.e., slope or height) and the denudation rates (mm/ky)*
+*Fig 2. The relationship between the slope and the denudation rates (mm/ky)*
 
 We can see that both the slope and precipitation could increase the denudation rates, and reaches a 'steady state' after a certain point.
 
-In terms of implementation, I used the function form of $D = P * S$, where $D$ means denudation rates, $P$ means effects of precipitation while $S$ means effects of Slope. By doing so, we can consider both effects. Such formula structure is similar to RUSLE model, a widely used LEM (e.g., (thapa_spatial_2020)[@cite]).
+Therefore, we could use the function form of $D = P * S$, where $D$ means denudation rates, $P$ means effects of precipitation while $S$ means effects of Slope. By doing so, we can consider both effects. Such formula structure is similar to RUSLE model, a widely used LEM (e.g., (Thapa et al., 2020)[@https://doi.org/10.1016/j.ejrh.2020.100704]). We use [sigmoidal function](https://en.wikipedia.org/wiki/Sigmoid_function) to approximate the influence of P or S on D, by fitting the function with the observed data and rendering parameter a, b, c, d, e, f. These are impleted as ```empirical_denudation```. For more information please refer to the documentation in NonCarboKitten.
```

Comment:
1. I can fix those references for you.
2. Please put mathematical symbols between dollar signs: `... the influence of $P$ or $S$ on $D$, by fitting ...` is rendered as "... the influence of $P$ or $S$ on $D$, by fitting ...".
3. Abbreviations should be explained when first used. In this case you may get away with "Large Eddy Model (LEM)".

Reply:
Hmmm, following that reference leaves me at a paper that mentions neither LEM nor RUSLE.

Reply:
What is `NonCarboKitten`? Your secret project?

Reply:
NonCarboKitten is here: https://github.com/MindTheGap-ERC/Non_carbokitten
I think there is some misunderstanding here, because @xyl96 used it a bit as a dump for different projects, including deriving the parameters in this equation, processing photos from Bahamas, a completely different code based on a discussion with Peter... @xyl96, please take a look at how repos are organized in our GitHub organization. Always assume other people will use this repo (even if just your teammates) so one repo = one purpose. In this case, you are referring to details of a regression model you have used to fit these parameters. This will go into a paper describing the erosion models you've developed. So that will be a repo with all code supporting that paper. Code that does not belong to that paper should be in different repos, reflecting its purpose. We can split it today together if you want.

Reply:
I think this has been resolved right now